src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -179,7 +179,7 @@ describe('HmctsGlobalHeaderComponent', () => {
     component.items = navItemsWithFindCaseRightAligned;
     await component.ngOnChanges(changesMock);
     fixture.detectChanges();
-    const findCase = fixture.debugElement.query(By.css('#find-case-link'));
+    const findCase = fixture.debugElement.query(By.css('.hmcts-search-toggle__button'));
```

Comment:
@johnbenjamin-hmcts - this change is already in the `feature/global-search` branch; you just need to update your PR to point to it.